### PR TITLE
ctb: Add echidna tests for portal

### PR DESCRIPTION
**Pull request by maurelian**
**Description**

Echidna tests for the portal.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -848,6 +848,10 @@ workflows:
           echidna_target: hashing
           requires:
             - bedrock-echidna-build
+      - bedrock-echidna-run:
+          echidna_target: portal
```
Comment (trianglesphere) [inline .circleci/config.yml:852]: name please

Comment (maurelian) [reply .circleci/config.yml:853]: fixed.

**Review by trianglesphere — APPROVED** [review]
thanks for adding the name.

**Review by tynes — APPROVED** [review]

**Comment by maurelian** [comment]
replaced by #4405.

**Pull request closed without merging**